code/game/objects/items/devices/flashlight.dm

```diff
@@ -51,7 +51,7 @@
 		if((HAS_TRAIT(user, TRAIT_CLUMSY) || user.getBrainLoss() >= 60) && prob(50))	//too dumb to use flashlight properly
 			return ..()	//just hit them in the head
 
-		if(!(istype(user, /mob/living/carbon/human) || SSticker) && SSticker.mode.name != "monkey")	//don't have dexterity
+		if(!istype(user, /mob/living/carbon/human))	//don't have dexterity
```

Comment:
Since it seems like monkey is a depreciated/non-existent gamemode, I just dropped it.